SoundRules.txt

```diff
@@ -1,25 +0,0 @@
-Wemade Mir2
-----------
```

Comment:
Please add this file back, it's a useful reference guide

Reply:
Sorry - must have missed it. Will add in.